.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.idea
```

Comment:
I just want to ignore my IDE specific config files. Nothing related to the code.